.ci/Jenkinsfile_cloud

```diff
@@ -3,18 +3,15 @@ library 'kibana-pipeline-library'
 def loadPipeline
 def MAVEN_IMAGE =  'maven:3.6.3-openjdk-8-slim'
 def MAX_TEST_RUNS = 10
-def MAX_DEPLOYMENTS = 3
-def threads = [:]
 
 pipeline {
     agent { label 'docker && ubuntu-tests-l' }
     options {
-        timeout(time: 90, unit: 'MINUTES')
+        timeout(time: 120, unit: 'MINUTES')
     }
     parameters {
         string(name: 'NUMBER_TEST_RUNS', defaultValue: '1', description: 'Running the same scenario up to 10 times')
-        string(name: 'NUMBER_DEPLOYMENTS', defaultValue: '1', description: 'Running on parallel deployments, up to 3')
```

Comment:
this logic is no longer needed: we won't run tests in parallel on cloud in the near future

Reply:
Does removing this line drop `NUMBER_DEPLOYMENTS` from the jenkins ui here: https://user-images.githubusercontent.com/10977896/116994413-f8157000-acd8-11eb-98bd-7b46c654e33a.png

Reply:
Yes, exactly. It wouldn't be there as soon we run changes on Jenkins